{{ cookiecutter.dir_name }}/src/android/PythonActivity.java

```diff
@@ -3,9 +3,10 @@
 import android.app.Activity;
 import android.os.Bundle;
 import android.util.Log;
+import android.support.v7.app.AppCompatActivity;
 
 
-public class PythonActivity extends Activity {
+public class PythonActivity extends AppCompatActivity {
```

Comment:
Looking at the docs, I'm fairly certain we'll be using ActionBar functionality for toga, so yeah - this is definitely required :-)